src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravVurderingQueries.kt

```diff
@@ -138,6 +139,21 @@ private fun ResultSet.toPAktivitetskravVurdering(): PAktivitetskravVurdering = P
     createdBy = getString("created_by"),
     status = getString("status"),
     beskrivelse = getString("beskrivelse"),
-    arsaker = getString("arsaker").split(",").map(String::trim).filter(String::isNotEmpty),
+    arsaker = getString("arsaker").split(",").map(String::trim).filter(String::isNotEmpty).map { Arsak.valueOf(it) },
     frist = getDate("frist")?.toLocalDate(),
 )
+
+private fun VurderingArsak.toDBString(): String =
```

Comment:
Tror likevel jeg hadde foretrukket en `toString` på VurderArsak i stedet for å gjenta dette i `toDBString` og `toDTOString` :sweat_smile: 
Og så kunne man kanskje i stedet for å hardkode strengene brukt Arsak.OPPFOLGINGSPLAN_ARBEIDSGIVER.name osv.